templates/bet/handlers/bet.ts

```
@@ -0,0 +1,94 @@
+                        storage.opponentAccepted,
+                        interactorRole
+                    ),
+                    handler: 'counterpartyAccepted',
+                }
+            }
+        } else if (interactorRole === 'arbitrator') {
+            if (storage.opponentAccepted && !storage.winner) {
+                return {
+                    buttons: [
+                        { label: 'Back' },
+                        { label: `Owner Wins` },
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)